tox.ini

```diff
@@ -40,7 +40,7 @@ deps =
     {[unitbase]deps}
     -r{toxinidir}/requirements_dev.txt
 install_command =
-    py{36,37,38,39,launch}: pip install --extra-index-url https://download.pytorch.org/whl/cpu {opts} {packages}
+    py{36,37,38,39,launch}: pip install --timeout 600 --extra-index-url https://download.pytorch.org/whl/cpu {opts} {packages}
```

Comment:
why install takes so much time???

Reply:
TCP socket timeouts, see the PR description for the two failures that these two fixes address.
we should considering looking into creating a local pypi cache eventually

Reply:
got it! Thanks!